boot/zephyr/Kconfig

```diff
@@ -74,18 +47,15 @@ config BOOT_USE_CC310
 
 config BOOT_USE_NRF_CC310_BL
 	bool
-	select NRF_CC310_BL
+	default n
```

Comment:
Why is there a `default n` here ?

Reply:
this patch reverts this line to its upstream state